src/pages/ReimbursementAccount/BusinessInfo/BusinessInfo.tsx

```diff
@@ -0,0 +1,144 @@
+import lodashPick from 'lodash/pick';
+import React, {useCallback, useMemo} from 'react';
+import {View} from 'react-native';
+import type {OnyxEntry} from 'react-native-onyx';
+import {withOnyx} from 'react-native-onyx';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import InteractiveStepSubHeader from '@components/InteractiveStepSubHeader';
+import ScreenWrapper from '@components/ScreenWrapper';
+import useLocalize from '@hooks/useLocalize';
+import useSubStep from '@hooks/useSubStep';
+import type {SubStepProps} from '@hooks/useSubStep/types';
+import useThemeStyles from '@hooks/useThemeStyles';
+import getInitialSubstepForBusinessInfo from '@pages/ReimbursementAccount/utils/getInitialSubstepForBusinessInfo';
+import getSubstepValues from '@pages/ReimbursementAccount/utils/getSubstepValues';
+import * as BankAccounts from '@userActions/BankAccounts';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type {ReimbursementAccount, ReimbursementAccountDraft} from '@src/types/onyx';
+import AddressBusiness from './substeps/AddressBusiness';
+import ConfirmationBusiness from './substeps/ConfirmationBusiness';
+import IncorporationDateBusiness from './substeps/IncorporationDateBusiness';
+import IncorporationStateBusiness from './substeps/IncorporationStateBusiness';
+import NameBusiness from './substeps/NameBusiness';
+import PhoneNumberBusiness from './substeps/PhoneNumberBusiness';
+import TaxIdBusiness from './substeps/TaxIdBusiness';
+import TypeBusiness from './substeps/TypeBusiness';
+import WebsiteBusiness from './substeps/WebsiteBusiness';
+
+type BusinessInfoOnyxProps = {
+    /** Reimbursement account from ONYX */
+    reimbursementAccount: OnyxEntry<ReimbursementAccount>;
+
+    /** The draft values of the bank account being setup */
+    reimbursementAccountDraft: OnyxEntry<ReimbursementAccountDraft>;
+};
+
+type BusinessInfoProps = BusinessInfoOnyxProps & {
+    /** The workspace policyID */
+    policyID: string;
+
+    /** Goes to the previous step */
+    onBackButtonPress: () => void;
+
+    /** Exits flow and goes back to the workspace initial page */
+    onCloseButtonPress: () => void;
+};
+
+const bodyContent: Array<React.ComponentType<SubStepProps>> = [
+    NameBusiness,
+    TaxIdBusiness,
+    WebsiteBusiness,
+    PhoneNumberBusiness,
+    AddressBusiness,
+    TypeBusiness,
+    IncorporationDateBusiness,
+    IncorporationStateBusiness,
+    ConfirmationBusiness,
+];
+
+const businessInfoStepKeys = CONST.BANK_ACCOUNT.BUSINESS_INFO_STEP.INPUT_KEY;
+
+function BusinessInfo({reimbursementAccount, reimbursementAccountDraft, policyID, onBackButtonPress, onCloseButtonPress}: BusinessInfoProps) {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+
+    const getBankAccountFields = useCallback(
+        (fieldNames: string[]) => ({
+            ...lodashPick(reimbursementAccount?.achData, ...fieldNames),
```

Comment:
NAB: Wdyt to get rid of lodashPick here? It should be doable with native JS

Reply:
@Swor71 WDYT?

Reply:
imho, I don't think it's worth rewriting it to native js just for that use case, unless there is some major push to remove lodash altogether from the project but it doesn't seem to be the case

Reply:
If it is important, please don't remove it. There are other places where it is present.